v2/datastream-to-spanner/terraform/samples/all-infrastructure/terraform.tfvars

```diff
@@ -0,0 +1,71 @@
+# Common Parameters
+common_params = {
+  project                         = "<YOUR_PROJECT_ID>"   # Replace with your GCP project ID
+  region                          = "<YOUR_GCP_REGION>"   # Replace with your desired GCP region
+  migration_id                    = "<YOUR_MIGRATION_ID>" # Will be used as a prefix for all resources, auto-generated if not specified
+  add_policies_to_service_account = "<TRUE/FALSE>"        # This will decide if roles will be attached to service accounts or not.
+}
+
+# Datastream Parameters
+datastream_params = {
+  source_connection_profile_id  = "source-mysql"   # Or provide a custom ID
+  target_connection_profile_id  = "target-gcs"     # Or provide a custom ID
+  target_gcs_bucket_name        = "live-migration" # Or provide a custom bucket name
```

Comment:
nit: if possible, it would be better to have a random id attached after these names. To ensure that repeated runs of this job do not overwrite existing config or lead to unexpected behaviour.

Reply:
It's already there. If the user does not provide a `migration_id`, one is auto-generated!